airbyte-cdk/python/unit_tests/sources/file_based/scenarios/scenario_builder.py

```diff
@@ -67,16 +70,15 @@ def __init__(
 
     def validate(self) -> None:
         assert self.name
-        if not self.expected_catalog:
```

Comment:
I wanted to log something here if `not self.catalog` but I was afraid it could interfere with pytest log capture so I did not